### PR TITLE
INT-1824 feature flags codemod: true && x => x

### PR DESCRIPTION
**Pull request by grzpab**

### CODE REVIEW

File: remove-unused-feature-flags/index.ts [hunk 1]
```diff
@@ -102,7 +114,50 @@ export default function transform(
 			(_, index) => !indices.some((i) => index === i),
 		);
 
-		return true;
+		const scope = variableDeclarator._computeScope();
+
+		if (scope === null || !scope.path) {
```
Comment (hbjORbj) [inline remove-unused-feature-flags/index.ts:119]:
```suggestion
		if (!scope?.path) {
```

isn't this enough?

File: remove-unused-feature-flags/test.ts [hunk 2]
```diff
@@ -50,16 +54,24 @@ describe('remove-unused-feature-flags', function () {
 
 	it('should remove a feature flag check within Promise.all() (with options)', function () {
 		const INPUT = `
-        const [a, b] = await Promise.all([
+        const [b, a] = await Promise.all([
+			fnc('b'),
```
Comment (hbjORbj) [inline remove-unused-feature-flags/test.ts:58]: any reason that this comes before `Promise.resolve('a'),`?

Comment (grzpab) [reply remove-unused-feature-flags/test.ts:58]: yes, this is testing the removal of the variable names in different positions in the array

File: remove-unused-feature-flags/index.ts (hunk 1 above)
Comment (grzpab) [reply remove-unused-feature-flags/index.ts]: true, fixed

**Review by DmytroHryshyn — APPROVED** [review]

**Pull request merged**